packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -169,7 +169,7 @@ void createSettingsAarGradle(Directory androidDirectory) {
   final String currentFileContent = currentSettingsFile.readAsStringSync();
 
   final String newSettingsRelativeFile = globals.fs.path.relative(newSettingsFile.path);
-  final Status status = globals.logger.startProgress('✏️  Creating `$newSettingsRelativeFile`...');
+  final Status status = logger.startProgress('✏️  Creating `$newSettingsRelativeFile`...');
```

Comment:
Could you check that ✏️ is in the list to rewrite when emoji aren't supported? I think it's base/console.dart or base/terminal.dart?

Reply:
It wasn't. I added it to the list, and added tests for the rewriting since I couldn't find any

Reply:
Thanks!